### ISSUE TITLE
Why using LayerNorm in MLP?

### ISSUE THREAD

**Issue by pUmpKin-Co** [isaaccorley/detcon-pytorch#3]
Why using layernorm in MLP?

**Comment by isaaccorley** [comment]
I had some issues with the combination of SyncBatchNorm and EMA with distributed training so I just replaced it with LayerNorm as a workaround.

Nothing stopping you from changing it back to batch norm though.

**Comment by pUmpKin-Co** [comment]
Thanks for your reply.In my tiny expermient, I found that layernorm was slghtly worse than batchnorm.So I asked the reason for using layernorm.
